cmd/bosun/expr/funcs.go

```diff
@@ -936,6 +943,23 @@ func Rename(e *State, T miniprofiler.Timer, series *Results, s string) (*Results
 	return series, nil
 }
 
+func AddTags(e *State, T miniprofiler.Timer, series *Results, s string) (*Results, error) {
+	for _, section := range strings.Split(s, ",") {
+		kv := strings.Split(section, "=")
```

Comment:
do none of our existing parsing routines handle this format already? I feel like we do this a lot.